.pipelines/pipeline.yaml

```diff
@@ -323,27 +323,7 @@ stages:
   #     clusterDefinitionCniTypeKey: "azureCNIURLWindows"
   #     clusterDefinitionCniBuildOS: "windows"
   #     clusterDefinitionCniBuildExt: ".zip"
-
-  - template: singletenancy/aks-engine/e2e-dualstack-job-template.yaml
```

Comment:
will need to remove these from the `cleanup` stage depends block

Reply:
good catch 👍

Reply:
done